src/ExtensionsSample/Samples/EasyTableSamples.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ExtensionsSample.Models;
+using Microsoft.Azure.WebJobs;
+using Microsoft.WindowsAzure.MobileServices;
+using WebJobs.Extensions.EasyTables;
+
+namespace ExtensionsSample
+{
+    public static class EasyTableSamples
+    {
+        public static void InsertItem([EasyTable] out Item newItem)
+        {
+            newItem = new Item()
+            {
+                Id = Guid.NewGuid().ToString(),
+                Text = new Random().Next().ToString()
+            };
+        }
+
+        public static async Task EnqueueItemToProcess(
+            [TimerTrigger("00:01")] TimerInfo timer,
+            [EasyTable] IMobileServiceTableQuery<Item> itemQuery,
```

Comment:
I also don't know how that would work. Since Mobile Services has a javascript client, there may be something we can do but I don't expect it to work automatically.

---
In reply to: [54803457](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/45#discussion_r54803457) [](ancestors = 54803457)